### ISSUE TITLE
1:N - finalize and search in the same execution?

### ISSUE THREAD

**Issue by angelabr** [usnistgov/frvt#150]
According to the 1:N interface document, there are four phases: enrollment, finalization, probe creation and search.
Are these phases executed sequentially in the same execution? 
Or does testing stop at finalization and then make another call in a different execution to probe creation and search?
Basically I would like to know if memory after finalization will persist in the search phase.

Thank you.

**Comment by paddygr** [comment]
You cannot rely on memory persisting after finalization.

From: Angela BR ***@***.***>
Date: Friday, December 13, 2024 at 4:52 AM
To: usnistgov/frvt ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [usnistgov/frvt] 1:N - finalize and search in the same execution? (Issue #150)

According to the 1:N interface document, there are four phases: enrollment, finalization, probe creation and search.
Are these phases executed sequentially in the same execution?
Or does testing stop at finalization and then make another call in a different execution to probe creation and search?
Basically I would like to know if memory after finalization will persist in the search phase.

Thank you.
